plugin/pkg/scheduler/algorithmprovider/affinity/affinity.go

```diff
@@ -37,9 +37,9 @@ func affinityPredicates() util.StringSet {
 		"PodFitsResources",
 		"NoDiskConflict",
 		// Ensures that all pods within the same service are hosted on minions within the same region as defined by the "region" label
-		factory.RegisterFitPredicate("ServiceAffinity", algorithm.NewServiceAffinityPredicate(factory.PodLister, factory.ServiceLister, factory.MinionLister, []string{"region"})),
+		factory.RegisterFitPredicate("RegionAffinity", algorithm.NewServiceAffinityPredicate(factory.PodLister, factory.ServiceLister, factory.MinionLister, []string{"region"})),
```

Comment:
Since generic nested topological levels have been implemented, I would be wary of taking a step back and restricting the number of nested levels. 

And I agree, I will remove the affinity provider for now and just have affinity/anti-affinity at the different levels be generically defined in documentation.